plugins/polio/js/src/domains/VaccineModule/SupplyChain/Details/VaccineRequestForm/VaccineRequestForm.tsx

```diff
@@ -225,6 +225,14 @@ export const VaccineRequestForm: FunctionComponent<Props> = ({
                                 disabled={false}
                             />
                         </Grid>
+                        <Grid item xs={6} md={3}>
+                            <Field
+                                label={formatMessage(MESSAGES.targetPopulation)}
+                                name="vrf.target_population"
+                                component={NumberInput}
+                                disabled={false}
```

Comment:
is the disable not set false by default ?

Reply:
yes, I don't know why I did that